### PR TITLE
Create Javascript Objects Dynamically

### PR DESCRIPTION
**Pull request by BryteLitty**
In this project, my coding partner and I worked on the following:
- created a collection of books using an array of objects
- used local storage to store book collections
- add even lister to the remove back to remove objects when it gets clicked
- created a function to display data in the local storage when the page loads.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+//     // },
+//     // {
+//     //     title: "Atomic Habits",
+//     //     author: "James Clear",
+//     // },
+//     // {
+//     //     title: "No Sweetness Here",
+//     //     author: "Ama Atta-Aiddo",
+//     // },
+// ]
```
Comment (Nemwel-Boniface) [inline app.js]: - [ ] Kindly remove all commented out, unused. unwanted pieces of code from your code base. It makes it look messy and cluttered.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  return content;
+}
+
+function injectContent(arr) {
+  let books = '';
+
+  arr.forEach((object) => {
+    books += createCollection(object);
+  });
+  // console.log(books)
```
Comment (Nemwel-Boniface) [inline app.js]: - [ ] Kindly remove all commented out, unused. unwanted pieces of code from your code base. It makes it look messy and cluttered.

File: app.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  booksContainer.innerHTML = books;
+  // console.log(booksContainer.innerHTML)
```
Comment (Nemwel-Boniface) [inline app.js]: - [ ] Kindly remove all commented out, unused. unwanted pieces of code from your code base. It makes it look messy and cluttered.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @BryteLitty and @lugard1    :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**